### PR TITLE
feat(iam): enabling both local and federated accounts (#10496)

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `master` 
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | ENA-68 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [ ] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

ENA68 allow users to have both local users and federated users in the same time.
+ a lot of quicks fix
* Clean all error messages
* Display missing right when error is 403
* Fix display with users > federation > groups (was mixed before)
* Remove customer code that was confusing and push compatible nichandle

## Related

:flags: Translation

- [x]  CMT-6630

Same PR as https://github.com/ovh/manager/pull/10496

### CODE REVIEW

**Review by darsene — APPROVED** [review]

**Review by sachinrameshn — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/account/user/users/add/user-users-add.controller.js [hunk 1]
```diff
@@ -60,12 +58,20 @@ export default class UserAccountUsersAddCtrl {
...
+              'user_users_add_error_message',
+            )} ${this.$translate.instant('user_need_rights_message')} ${
+              err.data.details.unauthorizedActionsByIAM
+            }`,
+            'userUsers',
+          );
+        }
+        return this.alerter.error(
+          `${this.$translate.instant('user_users_add_error_message')} ${
+            err.data.message
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/user/users/add/user-users-add.controller.js:73]:
```suggestion
            err?.data?.message
```

File: packages/manager/apps/dedicated/client/app/account/user/users/delete/user-users-delete.controller.js [hunk 2]
```diff
@@ -29,12 +27,20 @@ export default class UserAccountUsersDeleteCtrl {
...
+              'user_users_delete_error_message',
+            )} ${this.$translate.instant('user_need_rights_message')} ${
+              err.data.details.unauthorizedActionsByIAM
+            }`,
+            'userUsers',
+          );
+        }
+        return this.alerter.error(
+          `${this.$translate.instant('user_users_delete_error_message')} ${
+            err.data.message
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/user/users/delete/user-users-delete.controller.js:42]:
```suggestion
            err?.data?.message
```

File: packages/manager/apps/dedicated/client/app/account/user/users/disable/user-users-disable.controller.js [hunk 3]
```diff
@@ -29,12 +27,20 @@ export default class UserAccountUsersDisableCtrl {
...
+              'user_users_disable_error_message',
+            )} ${this.$translate.instant('user_need_rights_message')} ${
+              err.data.details.unauthorizedActionsByIAM
+            }`,
+            'userUsers',
+          );
+        }
+        return this.alerter.error(
+          `${this.$translate.instant('user_users_disable_error_message')} ${
+            err.data.message
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/user/users/disable/user-users-disable.controller.js:42]:
```suggestion
            err?.data?.message
```

**Review by anooparveti — CHANGES_REQUESTED** [review]
And in all other files

**Pull request merged**